joomla/joomla-cms
# [4.0] Global config media query fix

Pull Request for Issue #18595 .

### Summary of Changes
Custom media query breakpoint for the global config


### Testing Instructions
Apply PR and navigate to global config. Resize browser width and ensure fields never overlap. See original issue screenshot.


### Expected result



### Actual result



### Documentation Changes Required
